dimod/constrained/constrained.py

```diff
@@ -510,7 +510,8 @@ def add_discrete(self, data, *args, **kwargs) -> Hashable:
     def add_discrete_from_comparison(self,
                                      comp: Comparison,
                                      label: Optional[Hashable] = None,
-                                     copy: bool = True) -> Hashable:
+                                     copy: bool = True,
+                                     do_var_check: bool = True) -> Hashable:
```

Comment:
I think `check_valid` or `check_overlaps` would be a better name, if we want to keep the "direction" of having it default to `True`.

Reply:
`check_overlaps` is a great name! I'll stick with that